docs/examples/tf/tf_wandb_hydra/README.md

```diff
@@ -0,0 +1,78 @@
+# [Weights&Biases - Hydra](https://github.com/jean-zay-users/jean-zay-doc/tree/master/docs/examples/tf/tf_wandb_hydra)
+
```

Comment:
It could be great to add a little introduction here! :+1:

Reply:
Agreed, wdyt about what I wrote?

Reply:
It is great! Thank you

Reply:
Thanks for your review !